src/portal/lib/src/vulnerability-scanning/result-grid.component.ts

```diff
@@ -44,6 +51,29 @@ export class ResultGridComponent implements OnInit {
             this.loadResults(this.repositoryId, this.tagId);
         });
     }
+    getLevel(v: VulnerabilityItem) {
+        if (v) {
```

Comment:
define the return value in a const or enum.

Reply:
Fixed